MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/Tomography/ITomographyIfaceView.h

```diff
@@ -288,9 +269,9 @@ class ITomographyIfaceView {
   /**
    * Show a tool specific configuration dialog for the user to set it up
    *
-   * @param name human readable name of the tool, as a string
+   * @param dialog The pointer to the current dialog
    */
-  virtual void showToolConfig(const std::string &name) = 0;
+  virtual void showToolConfig(TomoToolConfigDialogBase *dialog) = 0;
```

Comment:
I've reverted this change in the latest commit so it can be consistent with the presenter. All the pointers are now `* const`, but I am not sure if changing all the pointers to reference would be desirable